spotbugs-exclude.xml

```diff
@@ -0,0 +1,76 @@
+<FindBugsFilter>
+    <!--Ignoring checking for examples-->
+    <Match>
+        <Package name="~io\.dapr\.examples.*"/>
+    </Match>
+
+    <!-- Metadata is an unmodifiable map -->
```

Comment:
We want to tackle these as a separate issue. Thanks for updating FindBugs too.

Reply:
https://github.com/dapr/java-sdk/pull/907 I did a PR dedicated to this. I included here because it was the only way to get a success status by running `mvn install` in the parent module.

Reply:
Rollback-ed these changes